cspell-custom-words.txt

```diff
@@ -0,0 +1,194 @@
+Acala
```

Comment:
When I had vs code add to dictionary, it rearranged the words to have the lower case before uppercase of the same spelling.

Reply:
I just used `sort --ignore-case` from the command line. Would you rather I generate this dictionary with vscode to do it the same?

Reply:
Redid the sorting without `--ignore-case` in 75c4c3ad5c830b749d3223ee1bdd1ee47e716d50

Reply:
Confirmed this was resolved with 8d6b02670c54f6ef000dceaae5b500047282d602